docs/en_US/Compressor/Pruner.md

```diff
@@ -342,14 +372,21 @@ PyTorch code
 ```python
 pruner.update_epoch(epoch)
 ```
-You can view example for more information
+You can view [example](https://github.com/microsoft/nni/blob/master/examples/model_compress/model_prune_torch.py) for more information.
 
 #### User configuration for AGP Pruner
-* **initial_sparsity:** This is to specify the sparsity when compressor starts to compress
-* **final_sparsity:** This is to specify the sparsity when compressor finishes to compress
-* **start_epoch:** This is to specify the epoch number when compressor starts to compress, default start from epoch 0
-* **end_epoch:** This is to specify the epoch number when compressor finishes to compress
-* **frequency:** This is to specify every *frequency* number epochs compressor compress once, default frequency=1
+
+##### PyTorch
+
+```eval_rst
+..  autoclass:: nni.compression.torch.AGP_Pruner
```

Comment:
could you help change all `AGP_Pruner` to `AGPPruner`? this pruner's name is different from all others...

Reply:
fixed